### DISCUSSION TITLE
Updated to the Feb version of HA and my Emporia YAML won't compile

### DISCUSSION THREAD

**Discussion by Aston77** [emporia-vue-local/esphome#123] [Q&A] [answered]
My other ESP32 devices update just fine but I get the below error on both of my Emporia Vue2.

Is this a problem with something on my end?

<img width="574" alt="image" src="https://user-images.githubusercontent.com/78501124/219229834-a0097033-5d5f-45e8-aad5-84a8e9468467.png">

**Comment by jordaosystems** [comment, marked as answer]
If you add mdns as below it will compile.

source: github://emporia-vue-local/esphome@dev
components: [ emporia_vue, mdns ]

**Reply by Aston77** [reply]
Thanks that did the trick. 

Just out of curiosity, any thoughts on why I all of a sudden needed to add the mdns snippet?

**Comment by jordaosystems** [comment]
It looks like Arduino specific code for esp32 mdns was removed causing the error.

**Comment by fisherbln** [comment]
Adding that code didn't fix the compile error for me. But that may be because the first write took out my wireless connection with the device. Seems I'm going to have to open my electrical box for the 4th time this month due to a poor firmware update by home assistant.

**Reply by clowrey** [reply]
Best practice for ESPhome is - don't update if it ain't broke ;)

**Reply by fisherbln** [reply]
I couldn't agree more. Unfortunately, home assistant framed it as an 'update' and I was thinking it was the add-on, not the firmware and clicked too fast. Ugh.

**Comment by flaviut** [comment]
Just merged in the new ESPHome version, should be good to go here.
